### PR TITLE
[refactor] LoRA tests

### PR DESCRIPTION
**Pull request by a-r-r-o-w**
# What does this PR do?

Minor cleanup of the lora tests

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@yiyixuxu @sayakpaul

### CODE REVIEW

**Review by sayakpaul — APPROVED** [review]
Looks good, thanks!

File: tests/lora/utils.py [hunk 1]
```diff
@@ -628,13 +600,9 @@ def test_simple_inference_with_text_denoiser_lora_save_load(self):
...
                 )
 
-            if self.unet_kwargs is not None:
-                pipe.unet.add_adapter(denoiser_lora_config)
-            else:
-                pipe.transformer.add_adapter(denoiser_lora_config)
-
-            denoiser_to_checked = pipe.unet if self.unet_kwargs is not None else pipe.transformer
-            self.assertTrue(check_if_lora_correctly_set(denoiser_to_checked), "Lora not correctly set in Unet")
+            denoiser = pipe.transformer if self.unet_kwargs is None else pipe.unet
```
Comment (yiyixuxu) [inline tests/lora/utils.py:603]: very nice!

**Review by yiyixuxu — APPROVED** [review]
very nice clean up!

**Comment by sayakpaul** [comment]
Thanks for the refactor!

**Pull request merged**